app/controllers/PaymentHistoryController.scala

```diff
@@ -44,7 +44,11 @@ class PaymentHistoryController @Inject()(val messagesApi: MessagesApi,
       if (isValidSearchYear(year) && appConfig.features.allowPaymentHistory()) {
         getFinancialTransactions(user, year).map {
           case Right(model) => Ok(views.html.payments.paymentHistory(model))
-          case Left(_)      => InternalServerError(views.html.errors.paymentsError())
+          case Left(_) => InternalServerError(views.html.errors.standardError(appConfig,
```

Comment:
Can you write a test for this _Left_ scenario please. It must have been missed when this was first implemented.